charts/api/templates/deployment-app-backend.yaml

```diff
@@ -216,15 +216,24 @@ spec:
                 name: {{ template "api.fullname" . }}-app-passport-keys
                 {{- end }}
                 key: oauth-private.key
-      {{- if .Values.app.gce.serviceAccountSecret }}
           volumeMounts:
+            {{- if .values.app.gce.serviceAccountSecret }}
             - name: "service-account-wbstack-api"
               mountPath: "/var/run/secret/cloud.google.com"
+            {{- end }}
+
+            - name: "mediawiki-configMap"
+              mountPath: "/config"
```

Comment:
I think probably this wants to go somewhere specific but I'm stalling a little on exactly where.

Perhaps into the Laravel config directory? (/var/www/html/config).

And it should definitely have a more meaningful name than `config` (maybe `mediawiki-backend-service-configmap.<extension>`. See https://github.com/wbstack/charts/pull/204/files#r2523682791